estate/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import estate_property, estate_property_type, estate_property_tags, estate_property_offer
```

Comment:
we prefer putting every import on a different line, so that if add something I directly know what has changed (here if you add an import, the whole will be considerer as changed). And if you want to not consider a file anymore, you can comment it easily if it's line by line =>
```
from . import estate_property
from . import estate_property_type
from . [...]
```